### PR TITLE
users_can_delete_their_shop

### PR DESCRIPTION
**Pull request by LevinGeorge**

### CODE REVIEW

File: app/controllers/shops_controller.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ def update
  	end
 
  	def destroy
+		@shop = Shop.find(params[:id])
```
Comment (danielpaul) [inline app/controllers/shops_controller.rb]: Use `current_user.shops.find...` to only allow permission to delete their own shop. Current code allows anyone to delete anyones shop. Use `current_user` relation for now as we don't have an authorisation system yet.

Comment (LevinGeorge) [inline app/controllers/shops_controller.rb]: Got it! :+1:

**Comment by danielpaul** [comment]
@LevinGeorge In the relation add `:dependent => :delete_all` so all the products in the shop are deleted too and not orphaned in the database just taking space. http://stackoverflow.com/questions/331394/can-i-set-up-cascade-deleting-in-rails

**Comment by danielpaul** [comment]
@LevinGeorge Is this a duplicate of https://github.com/SprintFire/marketplace/pull/27 ?

**Comment by LevinGeorge** [comment]
ehmm

**Pull request merged**